sprint/index.md

```diff
@@ -1,10 +1,73 @@
 # Sprint info
 
-% TODO: arreglar toctree y texto
-
 :::{toctree}
-:maxdept: 1
+:maxdepth: 1
+:hidden:
 
+self
 sprint-guide
 sprint-cheatsheet
+faq
+resources
+:::
+
+::::{div} sd-fs-5
+:::{tip}
+* **Wondering if the sprint is for you?** We have got just {ref}`the page <decision_tree>`
+* **Ready to contribute?** Go to {ref}`docstring_tutorial` page
+:::
+::::
+
+(sprint)=
+# Open Source Working Sessions
```

Comment:
I would use only a single level 1 heading. Maybe move this to be the top heading and keep the others as level 2 (maybe removing the `Sprint -` start they all have?)